app/src/main/java/com/github/sdp/mediato/data/ReviewInteractionDatabase.java

```diff
@@ -17,6 +20,20 @@ public class ReviewInteractionDatabase {
 
     public static FirebaseDatabase database = FirebaseDatabase.getInstance();
 
+    /**
+     * Adds a comment to a review.
+     * @param tarUsername the username of the user who made the review
+     * @param comment the comment
+     */
+    public static void commentReview(String tarUsername, Comment comment) {

```

Comment:
What does `tarUsername` mean?

Reply:
Target username. As specified in the java doc, it's the username of the user that made the review which is being commented (as opposed to refUsername - reference username which is the user that is currently logged in). The terminology is used in other database methods.